server/conn.go

```diff
@@ -78,6 +79,7 @@ type clientConn struct {
 	lastCmd      string            // latest sql query string, currently used for logging error.
 	ctx          QueryCtx          // an interface to execute sql statements.
 	attrs        map[string]string // attributes parsed from client handshake response, not used for now.
+	keepAlive    int64
```

Comment:
How about cmdCount?

Reply:
Add comment for it.

Reply:
This value is not `cmdCount`, just a flag: 
Somebody set it, if the connection is idle, the flag won't change.